assets/css/_cutout_overlay.scss

```diff
@@ -0,0 +1,44 @@
+.c-overlay-cutout {
+  z-index: 800; // Below controls, above map
```

Comment:
Really would prefer a better way to determine the layer z-index. Is there a sass map that already exists that might help?

Reply:
There's `$z-page-layout-context` in `app.scss`, that might be what you're looking for.